elastic/datadog_checks/elastic/elastic.py

```diff
@@ -508,26 +514,31 @@ def _process_custom_metric(
 
         if branch_value is not None:
             if xtype == "gauge":
-                self.gauge(metric_name, branch_value, tags=tags_to_submit, hostname=hostname)
+                self.log.debug("Submitting as gauge")
```

Comment:
Is this log needed ?

Note: the agent will also log metrics in `trace` log level.

Reply:
Removing.